src/components/TimelineEvent.js

```diff
@@ -1,10 +1,26 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
 
-const TimelineEvent = () => {
-  
-  return;
+
+const TimelineEvent = (props) => {
+  return (
+    <div className="TimelineEvent">
+      <ul>
+        <li>person: {props.person}</li>
+        <li>status: {props.status}</li>
+        <li>timestamp: <Timestamp time={props.timeStamp} /></li>
```

Comment:
See the CSS files for the class names

```suggestion
        <li className="event-person">person: {props.person}</li>
        <li className="event-status">status: {props.status}</li>
        <li className="event-time">timestamp: <Timestamp time={props.timeStamp} /></li>
```